app/models/pool/candidates.rb

```diff
@@ -46,19 +46,8 @@ def application_forms_in_the_pool
       .distinct
   end
 
-private
-
-  def filtered_application_forms
-    scope = curated_application_forms
-    scope = filter_by_subject(scope)
-    scope = filter_by_study_mode(scope)
-    scope = filter_by_course_type(scope)
-    scope = filter_by_right_to_work_or_study(scope)
-    filter_by_distance(scope)
-  end
-
-  def curated_application_forms
-    current_cycle_forms = ApplicationForm.current_cycle
+  def curated_application_forms(application_forms_scope = ApplicationForm.all)
+    current_cycle_forms = application_forms_scope.current_cycle
```

Comment:
Just thought that we can improve this a bit more.

```suggestion
      def curated_application_forms(application_forms_scope = ApplicationForm.current_cycle)
    current_cycle_forms = application_forms_scope
```

This way we always scope application forms to current cycle, when you don't pass in your specific application forms